ecology.yaml

```diff
@@ -462,7 +462,14 @@ tools:
   - name: fastq_combiner
     owner: devteam
     tool_panel_section_label: 'FASTA/FASTQ'
-
+    
+  # FROGS
+  - name: frogs
+    owner: frogs
```

Comment:
The owner is wrong and the section label is twice otherwise I think this is ready to go.

Reply:
If this is fixed I think we can merge it as soon as the new frogs version is in the toolshed.